handlers/tokenutil.go

```diff
@@ -40,18 +40,19 @@ func createTokenAsString(
 func getUserIdFromJwtToken(request *http.Request) (models.UserId, error) {
 	cookie, err := request.Cookie(cerealNotesCookieName)
 	if err != nil {
-		return -1, err
+		return 0, err
 	}
 
 	token, err := parseTokenFromString(cookie.Value)
 	if err != nil {
-		return -1, err
+		return 0, err
 	}
 
 	if claims, ok := token.Claims.(*JwtTokenClaim); ok && token.Valid {
 		return claims.UserId, nil
 	}
-	return -1, errors.New("Token was invalid or unreadable")
+
+	return 0, errors.New("Token was invalid or unreadable")
```

Comment:
I know this is actually code i made, but i'm realizing we should be consistent with errors and this should return a public named error, instead of a new error every time. This also makes it easier to check the type.

Reply:
Ya. I mean it makes error handling harder, since you can't check it a specific error was thrown.